build/build-buildimage-bases.sh

```diff
@@ -7,59 +7,70 @@
 # This script builds some base images that are needed for the build image:
 # - Python binaries
 # - PHP binaries
+# - Yarn package cache
 #
 
 set -e
 
 source __variables.sh
 
 IMAGE_TAG="${BUILD_NUMBER:-latest}"
+BUILD_IMAGE_PREFIX="$__REPO_DIR/images/build"
 
 # Clean artifacts
 mkdir -p `dirname $BUILD_IMAGE_BASES_ARTIFACTS_FILE`
 > $BUILD_IMAGE_BASES_ARTIFACTS_FILE
 
-# Build Python
-docker build -f $__REPO_DIR/images/build/python/prereqs/Dockerfile -t "python-build-prereqs" $__REPO_DIR
+case $1 in
```

Comment:
nit: put $1 in a local variable with a meaningful name, perhaps adding a validation if it is present when the script is run manually

Reply:
There's some form of validation in the last `case` clause - seems enough?